utils/getOffchainPrice.ts

```diff
@@ -0,0 +1,30 @@
+const fetch = require("node-fetch");
```

Comment:
This is not necessary as `fetch` is a native object in the browser API. That's why the name of the package is `node-fetch`, it's only for Node.js environments.

Reply:
Nice